docs/glossary.md

```diff
@@ -0,0 +1,42 @@
+# Key concepts
+
+This page provides help with the most common terminology used across Amadeus Self-service APIs.
```

Comment:
Hey Nico, do you think we could split them by categories? I feel that maybe people don't know always what they are looking for if they are not familiar with travel so it might help them to see them grouped by

Reply:
Hi Anna, sure, we can. Shall we adhere to the main API catalogue categories (Air, Hotel etc.)?

Reply:
Yes :)